lib/Config/Populator.php

```diff
@@ -689,6 +690,22 @@ private function updateSentUnsubscribeLinksToInstantUnsubscribeLinks() {
     ));
   }
 
+  private function pauseTasksForPausedNewsletters() {
+    if (version_compare($this->settings->get('db_version', '3.60.5'), '3.60.4', '>')) {
+      return;
+    }
+    global $wpdb;
+    $wpdb->query(
+      'UPDATE `' . ScheduledTask::$_table . '` t ' .
+      'JOIN `' . SendingQueue::$_table . '` q ON t.`id` = q.`task_id` ' .
+      'JOIN `' . Newsletter::$_table . '` n ON n.`id` = q.`newsletter_id` ' .
+      'SET t.`status` = "' . ScheduledTask::STATUS_PAUSED . '" ' .
+      'WHERE ' .
+      ' t.`status` = "' . ScheduledTask::STATUS_SCHEDULED . '" ' .
+      ' AND n.`status` = "' . Newsletter::STATUS_DRAFT . '" '
+    );
```

Comment:
It would be nicer to obtain table names using Doctrine and also use constants from entities instead of the old models so that we don't add more old models usage.
Here is how to get [table name from entity manager](https://github.com/mailpoet/mailpoet/blob/080a8221356019c513a80326c3789ab6d91ff56d/lib/Newsletter/NewslettersRepository.php#L221).

Reply:
If we inject the entityManager we could also use doctrine query instead of wpdb